nservicebus/azure-service-bus/multiple-namespaces-support.md

```diff
@@ -52,7 +52,7 @@ Exactly two namespaces have to be configured when using `FailOverNamespacePartit
 
 ## Cross namespace routing
 
-NServiceBus allows to specify destination addresses using an `"endpoint@physicallocation"` in various places such as the [Send](/nservicebus/messaging/send-a-message.md) and [Routing](/nservicebus/messaging/routing.md) API or the [MessageEndpointMappings](/nservicebus/messaging/message-owner.md). In this notation the `physicallocation` section represents the location where the endpoint's infrastructure is hosted, such as a machine name or a servicebus namespace.
+NServiceBus allows to specify destination addresses using an `"endpoint@physicallocation"` in various places such as the [Send](/nservicebus/messaging/send-a-message.md) and [Routing](/nservicebus/messaging/routing.md). In this notation the `physicallocation` section represents the location where the endpoint's infrastructure is hosted, such as a machine name or a servicebus namespace.
```

Comment:
I did not yet read the updated routing page, but since we explicitly disallow `RouteTo` parameters containing an "@" except for MEM, that would have to be an API declared by ASB similar to `.AtMachine` which should be documented here? Do we need to raise an issue on ASB about the requirements to support code first API for multi-namespace support (similar to the schema support issue @sqltransport)?